lib/pipeline-stack.ts

```diff
@@ -53,6 +54,9 @@ export class PipelineStack extends cdk.Stack {
     };
 
     const beta = createStage(scope, betaEnv, 'beta_a1_UsWest2', true);
+    // Creating the aspect here gives us the ability to add an adpect per application stage
+    scope.node.applyAspect(new CanaryAspect(scope, beta));
```

Comment:
Doesn't this create a single canary(stack) for ALL copies of the application?

Does that seem right to you?

When/how is the additional stack deployed?